### PR TITLE
[incubator/local-volume-provisioner] Add local-volume-provisioner chart in incubator folder

### PR DESCRIPTION
**Pull request by cofyc**
#### What this PR does / why we need it:

Add local-volume-provisioner chart in [sig-storage-local-volume-provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm) in incubator folder first.

xref: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/35

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: incubator/local-volume-provisioner/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+name: local-volume-provisioner
+version: 2.3.2
```
Comment (cpanato) [inline incubator/local-volume-provisioner/Chart.yaml]: this can be 1.0.0

File: incubator/local-volume-provisioner/templates/namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{{- if .Values.common.createNamespace }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.common.namespace | quote }}
+  labels:
+    heritage: {{ .Release.Service | quote }}
```
Comment (cpanato) [inline incubator/local-volume-provisioner/templates/namespace.yaml]: please use the new labels
see: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata

**Review by cpanato — CHANGES_REQUESTED** [review]
see some initial comments

**Comment by cofyc** [comment]
Updated.

File: incubator/local-volume-provisioner/templates/pod-security-policy.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{{- if .Values.common.podSecurityPolicy -}}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ include "local-volume-provisioner.fullname" . }}-psp
+  labels:
+		app.kubernetes.io/name: {{ include "local-volume-provisioner.name" . }}
```
Comment (cpanato) [inline incubator/local-volume-provisioner/templates/pod-security-policy.yaml]: fix the spaces here

Comment (cofyc) [reply incubator/local-volume-provisioner/templates/pod-security-policy.yaml]: ah, sorry about my stupid mistakes, fixed

**Comment by cofyc** [comment]
@cpanato
I also added some CI values files. How can I run e2e tests locally?

**Comment by kfox1111** [comment]
This seems useful still. Can we open this back up?

**Pull request closed without merging**